_quarto.yml

```diff
@@ -59,15 +62,10 @@ format:
       opts_chunk:
         R.options:
           width: 64
-    include-in-header: 
-      text: |
-        \usepackage{fvextra}
-        \DefineVerbatimEnvironment{Highlighting}{Verbatim}{breaklines,commandchars=\\\{\}}
-        \frontmatter
+    include-in-header: latex/in_header.tex
```

Comment:
Not necessary, but seeing as we'd done this for the others, I thought I'd keep this consistent